notify/impl.go

```diff
@@ -613,15 +619,24 @@ func (n *PagerDuty) notifyV2(
 
 	var buf bytes.Buffer
 	if err := json.NewEncoder(&buf).Encode(msg); err != nil {
-		return false, err
+		return false, fmt.Errorf("failed to encode PagerDuty v2 message: %v", err)
 	}
 
 	resp, err := post(ctx, c, n.conf.URL.String(), contentTypeJSON, &buf)
 	if err != nil {
-		return true, err
+		return true, fmt.Errorf("failed to post message to PagerDuty: %v", err)
 	}
 	defer resp.Body.Close()
 
+	// See: https://v2.developer.pagerduty.com/docs/events-api-v2#api-response-codes--retry-logic
+	if resp.StatusCode == http.StatusBadRequest {
+		body, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			return false, fmt.Errorf("failed to read error response from PagerDuty(status: %d): %v", resp.StatusCode, err)
```

Comment:
Add a space between `PagerDuty` and `(`.

Reply:
Done

Reply:
Hmm, it is still there...

Reply:
Sorry I somehow missed that I had this twice, in the log statement and in the error message... . Now it should really be fixed :)